src/bundler.ts

```diff
@@ -126,26 +126,24 @@ class Bundler {
   resolveBundleUrl(
       url: string,
       bundle: AssignedBundle,
-      manifest: BundleManifest): boolean|string {
+      manifest: BundleManifest): string|false {
     const targetBundle = manifest.getBundleForFile(url);
     if (!targetBundle || !targetBundle.url) {
       return false;
     }
+    for (const exclude of this._excludes) {
+      if (url.search(exclude) > -1) {
```

Comment:
`String.prototype.search` takes a RegExp, and will convert a string into a RegExp via `new RexExp`. That could lead to some funky behavior with some strings.